scripts/test/shared.py

```diff
@@ -101,6 +107,18 @@ def warn(text):
 
 # setup
 
+if not options.binaryen_lib:
+    if os.environ.get('BINARYEN_ROOT'):
+        if os.path.isdir(os.path.join(os.environ.get('BINARYEN_ROOT'), 'lib')):
+            options.binaryen_lib = os.path.join(
+                os.environ.get('BINARYEN_ROOT'), 'lib')
+        else:
+            options.binaryen_lib = os.environ.get('BINARYEN_ROOT')
+    else:
+        options.binaryen_lib = 'lib'
+
+options.binaryen_lib = os.path.normpath(os.path.abspath(options.binaryen_lib))
```

Comment:
Then you can just skip all this an just do:

```
if not options.binaryen_lib:
  options.binary_lib = ..realtive_to_binaryeh_bin..
```

Down below 133.

Reply:
Yes, thanks for the suggestion.